### ISSUE TITLE
Real differences with GetX

### ISSUE THREAD

**Issue by asterd** [Aniketkhote/refreshed#2]
Hi, I love this approach and I Hope you can go ahead with this package since original GetX has a very big lack in support and this Is a big problem in modern development environment.
Now, I Hope you can explain in a Little more details what are the differences between this version and the original One:

- It Is based on GetX 5 or 4.x?
- how many original issues were resolved?

I Hope also to find some times to support you.
Best regards and thanks a lot for this try!

**Comment by aniketkhote9** [comment]
Hi there,

Thank you for your interest and support! We truly appreciate your enthusiasm for this package. Allow me to address your questions:

Version Compatibility: This version of the package is based on GetX 4.x.

Resolved Issues: We are actively working on resolving existing compatibility issues and fixing deprecated Flutter methods. While we cannot provide an exact count of how many issues have been resolved, rest assured that we are dedicated to enhancing the reliability and stability of this package with each update.

Your willingness to support us is incredibly valuable, and we welcome any contributions or feedback you may have. Together, we can make this package even better suited for modern development environments.

Best regards, and thank you once again for your support!

**Comment by DevTello** [comment]
Hi, @aniketkhote9, thank you for your efforts regarding this package!
1) Why did you fork from GetX 4.x and not 5?
2) Do you have kind of a roadmap?

**Comment by Aniketkhote** [comment]
1. Apologies for the confusion earlier. I misspoke. The package is based on the latest master branch of GetX, ensuring that we're aligned with the most recent developments and improvements.

2. In terms of a roadmap, our main focus is to fix existing bugs while maintaining compatibility with the latest Flutter and Dart versions. We're also prioritizing the decoupling of irrelevant modules to streamline the package. For example, we've decoupled GetConnect and published it separately as Connectify. Additionally, we've removed other irrelevant methods and extensions, such as animations or padding extensions. You're welcome to give it a try using quickly or do_animate.
